### ISSUE TITLE
Validate `v0.10.1-rc2` in staging environment

### ISSUE THREAD

**Issue by frrist** [filecoin-project/lily#1006]
### Description
`v0.10.1-rc2` has been deployed to the staging environment. It needs to be validated before it can be pushed to production.
Deployment tracking here: https://github.com/filecoin-project/sentinel-infra/issues/278

### Acceptance criteria
- [x] All tables are being populated with data
- [x] https://github.com/filecoin-project/lily/issues/997 remains fixed
- [x] Notifier is successfully adding tasks to Redis
- [x] Workers are successfully consuming and processing tasks from Redis
- [x] Workers are capable of keeping up with the chain and the Redis queue zero tasks waiting for processing

**Comment by kasteph** [comment]
For the todo: All tables are being populated with data

## miner_locked_funds

```
select max(height)
from miner_locked_funds;
```

| prod | staging|
|------|------|
|1763738|1938100|

------

```
select count(*)
from miner_locked_funds
where height = 1938100;
```

| prod | staging|
|------|------|
|0|226|

## miner_sector_deals

```
select max(height)
from miner_sector_deals;
```

| prod | staging|
|------|------|
|1938101|1938104|

------

```
select count(*)
from miner_sector_deals
where height = 1938100;
```

| prod | staging|
|------|------|
|7|14|

## miner_sector_infos_v7

```
select max(height)
from miner_sector_infos_v7;
```

| prod | staging|
|------|------|
|1938107|1938107|

------

```
select count(*)
from miner_sector_infos_v7
where height = 1938100;
```

| prod | staging|
|------|------|
|76|281|


---

I'm happy to do a set difference if you deem necessary @frrist. It's just not as easy as running these simple queries because I need to export the tables from the database and then import them into a local db with duplicate table schemas.

**Comment by frrist** [comment]
Thanks, @kasteph - a set difference won't be needed. This confirms we have addressed #997

**Comment by frrist** [comment]
cut release closing: https://github.com/filecoin-project/lily/releases/tag/v0.10.1
